source/Calamari/Commands/DeployPackageCommand.cs

```diff
@@ -73,10 +73,8 @@ public override int Execute(string[] commandLineArguments)
 
             var conventions = new List<IConvention>
             {
-                new ContributeEnvironmentVariablesConvention(),
                 new ContributePreviousInstallationConvention(journal),
                 new ContributePreviousSuccessfulInstallationConvention(journal),
```

Comment:
This change does mean the variables contributed by this convention are not logged, however #510 re-instates this behaviour